### DISCUSSION TITLE
How to add 4,6,7,9 key to 1.6.2 kade engine?

### DISCUSSION THREAD

**Discussion by glowsoony** [KadeArchive/Kade-Engine#2793] [Q&A] [answered]
I find at hard as 1.5.3 and older versions have different types of code from 1.6 and up and i always get errors.

**Comment by ghost** [comment, marked as answer]
I'm not sure, but try looking at the source code of some mods.
Like this one.
Shaggy X Matt: https://github.com/GithubSPerez/shaggy-matt

**Comment by RobGames678** [comment]
Look, I Don't Know Anything From This

**Reply by ghost** [reply]
THATS A NOTIFICATION WHEN I SENT A MESSAGE IN THIS THREAD
